src/python/dxpy/toolkit_version.py

```diff
@@ -1 +1 @@
-version = '0.304.0'
+version = '0.304.1'
```

Comment:
Afair this will be updated automatically at the release.

Reply:
removed this change